test/migration_generator.rdoctest

```diff
@@ -448,13 +452,13 @@ The migration generator respects the `set_table_name` declaration, although as b
        self.table_name="ads"
        fields do
          title :string, :default => "Untitled", limit: 255, null: true
-         body: text, limit: 0xffff
+         body :text, limit: 0xffff, null: true
```

Comment:
Weird...how could `body:` have ever passed?!

Reply:
Yea this looked to be some really bad copy/paste that was going on and we never had the tests running in CI until now.

Reply:
Ah, so probably the test passed in the author's branch...but we broke it somehow on our fork.